nestjs/src/courses/mentors/mentors.controller.ts

```diff
@@ -20,6 +20,7 @@ export class MentorsController {
       throw new ForbiddenException();
     }
     const items = await this.mentorsService.getStudents(mentorId);
-    return items.map(item => new MentorStudentDto(item));
+    const courseName = await this.mentorsService.getCourseName(mentorId);
```

Comment:
Looks like this can be done in parallel, let's use Promise.all here :)

Reply:
Please, check the changes